database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
+  s_subscription_object jsonb NOT NULL,
+  CHECK (s_subscription_object->>'id' = s_id)
```

Comment:
For me the question is "do we really need to have s_id as a separate table column?" as we anyway store the whole object to json. Probably it makes sense for indexing purposes. And I don't see a good reason not to store it in json.
And also that will make this functionality consistent with event-types table (where we store the name as a separate table column and in json object)